README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
```

Comment:
@jcscottiii This raises the question, though: should we move this to be part of `cf-hello-worlds` repo?